test/h2spec/client.go

```diff
@@ -28,7 +31,6 @@ func NewClientCommand() *cobra.Command {
 	}
 
 	cmd.SilenceUsage = true
-	cmd.SilenceErrors = true
```

Comment:
This allows timeout errors to be reported, but not as a returned error value; the h2spec code just prints these directly: https://github.com/openshift/cluster-ingress-operator//blob/master/vendor/github.com/spf13/cobra/command.go#L986.